tests/python/test_ggui.py

```diff
@@ -31,6 +31,20 @@ def write_temp_image(window):
         pass
 
 
+def write_temp_image_array(img, file=None):
+    if img.shape[0] % 2 != 0:
```

Comment:
OOC: why clipping?

Reply:
just to keep consitant with https://github.com/taichi-dev/taichi/blob/367dc7df80feff6976fe96c3380e255b230639d5/python/taichi/tools/video.py#L116, but clipping is not used here cause we always use even number.